packages/fondue/src/components/SegmentedControls/SegmentedControls.tsx

```diff
@@ -142,15 +146,24 @@ export const SegmentedControls = ({
             <SegmentedControlsItem
                 id={id}
                 item={item}
-                disabled={disabled}
+                disabled={item.disabled ?? disabled}
                 radioGroupState={radioGroupState}
                 ref={(el) => (itemsRef.current[index] = el)}
                 key={`fondue-segmented-controls-${id}-item-${item.id}`}
                 size={size}
             />
         ));
     }, [items, id, disabled, radioGroupState, size]);
-    const selectedIndex = items.findIndex((item) => item.id === radioGroupState.selectedValue);
+
+    const selectedIndex = items.findIndex((item, index) => {
+        const isActiveItem = item.id === radioGroupState.selectedValue;
+        if (item.disabled && isActiveItem) {
+            const nextIndex = index + 1;
+            radioGroupState.setSelectedValue(items[nextIndex].id);
```

Comment:
Goes out of bounds in case of the last item being `disabled`, also a disabled item should be able to be selected which makes the changes in this function not necessary 🤔

Reply:
yes i agree, this can be reverted to it's original function. i don't se why a disabled item couldn't be default